source/common/upstream/cluster_manager_impl.h

```diff
@@ -517,6 +518,7 @@ class ClusterManagerImpl : public ClusterManager,
       LoadBalancerPtr lb_;
       ClusterInfoConstSharedPtr cluster_info_;
       Http::AsyncClientImpl http_async_client_;
+      std::unique_ptr<Http::PersistentQuicInfo> quic_info_;
```

Comment:
Nit: I think the Envoy style guide uses aliased types for smart pointers:

```
Smart pointers are type aliased:
using FooPtr = std::unique_ptr<Foo>;
```
https://github.com/envoyproxy/envoy/blob/main/STYLE.md